.gitignore

```diff
@@ -134,194 +180,211 @@ DocProject/Help/html
+**/*.DesktopClient/GeneratedArtifacts
+**/*.DesktopClient/ModelManifest.xml
+**/*.Server/GeneratedArtifacts
+**/*.Server/ModelManifest.xml
+_Pvt_Extensions
 
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
+# Paket dependency manager
+.paket/paket.exe
+paket-files/
 
-# Coverage directory used by tools like istanbul
-coverage
+# FAKE - F# Make
+.fake/
 
-# nyc test coverage
-.nyc_output
+# CodeRush personal settings
+.cr/personal
 
-# Grunt intermediate storage (https://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# Bower dependency directory (https://bower.io/)
-bower_components
+# Python Tools for Visual Studio (PTVS)
+__pycache__/
+*.pyc
 
-# node-waf configuration
-.lock-wscript
+# Cake - Uncomment if you are using it
+# tools/**
+# !tools/packages.config
 
-# Compiled binary addons (https://nodejs.org/api/addons.html)
-build/Release
+# Tabs Studio
+*.tss
 
-# Dependency directories
-node_modules/
-jspm_packages/
+# Telerik's JustMock configuration file
+*.jmconfig
 
-# TypeScript v1 declaration files
-typings/
+# BizTalk build output
+*.btp.cs
+*.btm.cs
+*.odx.cs
+*.xsd.cs
 
-# Optional npm cache directory
-.npm
+# OpenCover UI analysis results
+OpenCover/
 
-# Optional eslint cache
-.eslintcache
+# Azure Stream Analytics local run output
+ASALocalRun/
 
-# Optional REPL history
-.node_repl_history
+# MSBuild Binary and Structured Log
+*.binlog
 
-# Output of 'npm pack'
-*.tgz
+# NVidia Nsight GPU debugger configuration file
+*.nvuser
 
-# Yarn Integrity file
-.yarn-integrity
+# MFractors (Xamarin productivity tool) working folder
+.mfractor/
 
-# dotenv environment variables file
-.env
+# Local History for Visual Studio
+.localhistory/
 
-# parcel-bundler cache (https://parceljs.org/)
-.cache
+# BeatPulse healthcheck temp database
+healthchecksdb
 
-# next.js build output
-.next
+# Backup folder for Package Reference Convert tool in Visual Studio 2017
+MigrationBackup/
 
-# nuxt.js build output
-.nuxt
+# Ionide (cross platform F# VS Code tools) working folder
+.ionide/
 
-# vuepress build output
-.vuepress/dist
+# Fody - auto-generated XML schema
+FodyWeavers.xsd
 
-# Serverless directories
-.serverless/
+# VS Code files for those working on multiple tools
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+*.code-workspace
 
-# FuseBox cache
-.fusebox/
+# Local History for Visual Studio Code
+.history/
 
-#DynamoDB Local files 
-.dynamodb/ 
+# Windows Installer files from build outputs
+*.cab
+*.msi
+*.msix
+*.msm
+*.msp
 
+# JetBrains Rider
 .idea/
+*.sln.iml
 
-##############
-/.vs
-/Basic/DotNet/SdsClientLibraries/SdsClientLibraries/.vs/config/applicationhost.config
-/Basic/DotNet/SdsRestApiCore/.vs/config/applicationhost.config
-.vscode/launch.json
-
-*.zip
-cov-int/
-junit/
-*.binlog
-junit/
-
-# Credential Config Files
-product-readiness.*
-appsettings.json
-config.ini
-config.js
-config.properties
+## Credential Config Files
```

Comment:
Did you mean to remove the entries for the `appsettings.json` and `product-readiness.*` files?

Reply:
added the P-R.* entry back. since eds doesnt do any auth, there's no appsettings file to ignore, so i removed that entry as well.

Reply:
Forgot this one didn't have an `appsettings.json`. Sounds good!